snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -100,32 +100,34 @@ init_config:
   ## it directly here.
   ## When using a profile present on the profiles directory of the
   ## configuration, you can directly pass the filename.
+  ## By default, it loads all the profiles installed in the python package and
```

Comment:
This seems a bit confusing.  When would a user benefit from uncommenting only certain profiles?

What if they had a custom profile they want to add, will they need to uncomment everything to still track those metrics too or just add their new one?

Reply:
You don't necessarily want to monitor all devices? By selecting a subset of profiles you can control that.

If they have a custom profile, they have 2 options:
 - drop it in confd. It will load it automatically
 - Change the profiles section, in which case they need to list all the ones they want.

Reply:
👍 The README should be updated to align with this change to make it clear.

Reply:
Done.